src/main/java/de/retest/recheck/ui/diff/ElementDifference.java

```diff
@@ -85,15 +85,23 @@ public Screenshot mark( final Screenshot screenshot ) {
 		return differences;
 	}
 
-	public List<AttributeDifference> getAttributeDifferences( final ShouldIgnore ignore ) {
+	public List<AttributeDifference> getAttributeDifferences( final ShouldIgnore input ) {
 		final List<AttributeDifference> differences = new ArrayList<>();
+		final ShouldIgnore ignore = input != null ? input : ShouldIgnore.IGNORE_NOTHING;
```

Comment:
Is it possible to just return `differences`If `input` is `null`?

`if ( ignore != null && ignore.shouldIgnoreElement( element ) ) {...}`